### PR TITLE
feat(user experience): implemented user experience with new modal banner and logic

### PR DESCRIPTION
**Pull request by milan-bc**
## Description (optional)
This PR implements new user experience modal

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.
more details here https://www.loom.com/share/db3d781a5fed4856b8fd9a2dde9eb018

image

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts [hunk 1]
```diff
@@ -790,6 +790,16 @@ type MessagesType = {
   'modal.createaccountlearn.info3': "You can read more information about Stellar's minimum balance requirement at"
   'modal.createaccountlearn.link': 'Stellar.org'
   'modal.createaccountlearn.title': 'Stellar mimimum balance requirement.'
+
+  'modal.complete_profile.note': 'Your steps towards owning the future.'
+  'modal.complete_profile.finish_settings': 'Finish setting up your Blockchian.com account and start buying crypto today.'
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts:795]: typo in blockchain

File: packages/blockchain-wallet-v4-frontend/src/data/goals/sagas.ts [hunk 2]
```diff
@@ -736,7 +736,11 @@ export default ({ api, coreSagas, networks }) => {
       const sddEligible = yield call(api.fetchSDDEligible)
       // show SDD flow for eligible country
       if (sddEligible.eligible) {
-        return yield put(actions.components.buySell.showModal({ origin: 'WelcomeModal' }))
+        // return yield put(actions.components.buySell.showModal({ origin: 'WelcomeModal' }))
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/data/goals/sagas.ts:739]: remove or feature flag?

**Review by schnogz — CHANGES_REQUESTED** [review]
a few comments above.  Also please move the new modal into the `Onboarding` folder within the modals folder

File: packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts (hunk 1 above)
Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts:795]: Heh I reported this to design but I forgot to update it here, eagle eye!

File: packages/blockchain-wallet-v4-frontend/src/data/goals/sagas.ts (hunk 2 above)
Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/data/goals/sagas.ts:739]: adding a feature flag to this entire functionality

**Pull request merged**